.github/actions/win-sign-action/action.yml

```
@@ -0,0 +1,137 @@
+                    $jobs | Stop-Job | Remove-Job
+                    Write-Host "❌ Signing failed for $($result.FilePath) with exit code: $($result.ExitCode)"
+                    exit 1
+                } else {
+                    $successCount++
+                    Write-Host "  ✅ Successfully signed $($result.FilePath)"
+                }
+            }
+            # Run signing in a job
+            $job = Start-Job -ScriptBlock {
+                    param($signerPath, $username, $password, $signDescription, $signUrl, $filePath)
+                    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)